packages/cosmos_ui_components/lib/utils/date_formatter.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:intl/intl.dart';
+
+String formatDate(DateTime date, DateFormatEnum dateFormat) => DateFormat(getDateFormat[dateFormat]).format(date);
+
+abstract class DateFormats {
+  static const String UIShortDate = 'MMM d';
+  static const String groupByMonthYear = 'MMMM-yyyy';
+}
+
+enum DateFormatEnum { ShortUIDateDay, GroupByMonthYear }
```

Comment:
lets add trailing commas wherever possible for readability.
Also, most of the enum cases in our code start with lowercase letter, lets stick with that